api/src/main/java/com/epam/pipeline/manager/preference/SystemPreferences.java

```diff
@@ -823,6 +823,13 @@
         SEARCH_GROUP,
         isNullOrValidJson(new TypeReference<List<StorageFileSearchMask>>() {}));
 
+    public static final ObjectPreference<List<StorageFileSearchMask>> STORAGE_SIZE_MASK_RULES = new ObjectPreference<>(
```

Comment:
Please name this preference `STORAGE_QUOTAS_SKIPPED_PATHS` `storage.quotas.skipped.paths` and let's move it to storage group

Reply:
Done